operate/common/src/main/java/io/camunda/operate/connect/OpensearchConnector.java

```diff
@@ -95,6 +96,7 @@ public OpensearchConnector(
   }
 
   @Bean
+  @Primary
```

Comment:
Are we creating a bean of this type somewhere else as well? Primary can squash multiple bean issues but can lead to wrong expected bean types and usually means the creation needs to be looked at closer.